Assets/TimerUI.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class TimerUI : MonoBehaviour
+{
+    public GameObject clockhand;
+
+    public TextMeshProUGUI timerText;
+
+    public TimeSettings timeSettings;
+
+    // Update is called once per frame
+    void Update()
+    {
+        int lastIncrement = (int)timeSettings.CurrentMaxTimeSeconds() / (int)timeSettings.SecondsPerIncrement();
+        // a percentage value between 0-1
+        float progress = timeSettings.SecondsIntoIncrement() / timeSettings.SecondsPerIncrement();
+        if (lastIncrement == timeSettings.NextIncrement())
```

Comment:
```suggestion
        if (lastIncrement >= timeSettings.NextIncrement())
```
Since Update() is ran per frame, there's a chance it'll skip over if we use a direct equals here.

Reply:
good catch!